### ISSUE TITLE
Weird message appearing in chat, then multiple people unable to move

### ISSUE THREAD

**Issue by Blithering** [d3athrow/vgstation13#14696]
![corners](https://cloud.githubusercontent.com/assets/15067109/25209633/4d5d5528-2574-11e7-8f52-2e0edb44f274.png)
A message which appeared to be a snippet of conversation appeared in chat as if it were an admin narration, immediately following which several people lost the ability to move. they could still be pushed around by others, so they weren't somehow anchored.

- [ ] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by clusterfack** [comment]
Uhm, that sound a lot like direct narrate. Did you check the logs?

**Comment by Blithering** [comment]
@clusterfack No instance of those words appear in the logs before ahelps about it, did notice 
``[18:38:53]DEBUG: Runtime in <b>action.dm</b>, line <b>53</b>: <b>Cannot modify null.moved.</b> <a href='?_src_=holder;viewruntime=[0x21098d27]'>[view]</a>``
just before the first people started talking about being stuck.

**Comment by SonixApache** [comment]
It happened a while ago too, @PJB3005 was there. A debug message in a light related file, I think.

**Comment by unid15** [comment]
Spooky 💀

**Comment by PJB3005** [comment]
Yes, I forgot to remove the debug messages when me and MSO were trying to figure out why the fuck the shuttles were having broken lighting while porting StonedMC.

The messages should only happen when `SSlighting` sleeps during its processing, which causes the master controller to break and makes `SSlighting` "forget" its current processing queue. There's no sleeps though.

I suspect that somehow, BYOND's CPU counter internally breaks, so it begins inserting random sleep calls into loops which causes `SSlighting` to explode. The movement issue is another symptom as I suspect BYOND fails to handle input and ticks smoothly anymore.

I've personally tried to fix it when this happened and the CPU counter got stuck at around ~65536, and it persisted through would restarts indicating that the entire process broke and needed a restart. Couldn't manage to crash it though and we just nagged pomf.

**Comment by PJB3005** [comment]
[`code/controllers/subsystem/lighting.dm#L72`](https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/controllers/subsystem/lighting.dm#L72) for the message.

**Comment by Blithering** [comment]
@PJB3005 Is there an open issue for that? If so, I'll close this one as a duplicate.

**Comment by PJB3005** [comment]
@Blithering AFAIK no.

**Comment by Inorien** [comment]
that debug line was commented out 5 months after this issue was made, in sept 2017
the commented out line was finally removed in september 2021
and now in 2025 im killing this ancient issue because it seems to no longer occur :)
